babel/README.md

```diff
@@ -0,0 +1,139 @@
+
+Back in your browser, delete all the content in the editing window (click into the editing area and CTRL-A to select all, then delete), then paste your latest text into the online editing window.
+
+If everything went correctly, you should just see your edits when you click the "Preview changes" tab just above the online editing window.
+
+<img src="img/github_edit_2.png">
+
+Any changed lines will be highlighted. Red highlights are deletions, green are additions.
+
+Scroll to the bottom of that web page to the "Commit changes section". If your update addresses a specific issue that was raised in the repo's "Issues" section, reference it by number in the optional extended description box using this format: "Fixes #1234" (using the actual issue number, obv). Select the option to create a new branch and submit your change as a pull request:
+
+<img src="img/github_edit_3.png">
+
+Follow the prompts to finish submitting your pull request.
+
+A reviewer on the Specter Desktop team will review your changes and (hopefully) accept the changes. Your contributions will be merged into the main Specter Desktop code repository. You will officially be a Specter Desktop open source contributor!
+
+
+
+## For Developers:
+### "Wrapping" text for translation
+All you have to do in your code is wrap each piece of English text with the `ugettext` shorthand `_()`:
+* Wrap jinja template text: `<p>Hello, world!</p>` becomes `<p>{{ _('Hello, world!') }}</p>`
+* Wrap python strings: `error="No device was selected."` becomes `error=_("No device was selected.")`
+* Use `.format()` to wrap strings with variable injections:
+    ```
+    mystr = f"My dad's name is {dad.name} and my name is {self.name}."
+    mystr = _("My dad's name is {} and my name is {}").format(dad.name, self.name)
+    ```
+
+    If there are a lot of variables to inject, placeholder names can be used:
+    ```
+    mystr = _("My dad's name is {dad_name} and my name is {my_name}").format(dad_name=dad.name, my_name=self.name)
+    ```
+
+
+### Rescanning for text that needs translations
+Re-generate the `messages.pot` file:
+```
+pybabel extract -F babel/babel.cfg -o babel/messages.pot .
+```
+This will rescan all wrapped text, picking up new strings as well as updating existings strings that have been edited.
+
+Then run `update`:
+```
+pybabel update -N -i babel/messages.pot -d src/cryptoadvance/specter/translations
+```
+_note: the `-N` flag prevents babel from trying to use fuzzy matching to re-use existing translations for new strings. The fuzzy matching does not seem to do what we would want so we keep it disabled._
+
+Any newly wrapped text strings will be added to each `messages.po` file. Altered strings will be flagged as needing review to see if the existing translations can still be used.
+
+Once the next round of translations is complete, recompile the results:
+```
+pybabel compile -d src/cryptoadvance/specter/translations
+```
+
+
+### Adding support for another language
```

Comment:
Here it should also mention you need to update the list of languages in the `config.py`

Reply:
Good catch. Updated!